README.md

```diff
@@ -52,38 +59,51 @@ async function updateDom() {
   element.appendChild(...);
   element.id = "...";
 
-  // Acquire a lock.
-  await element.displayLock.acquire();
+  // Acquire a lock, and allow it to be unlocked by the user agent when
+  // necessary.
+  await element.displayLock.acquire({ activatable: true });
 
   // Append the element to the DOM.
   document.body.appendChild(element);
 
-  // Now we can update the element, causing co-operative updates. After that
-  // resolves, we can commit the element making it visible to the user.
-  element.displayLock.update().then(() => {
+  // Now even if the element is appended to the tree we can do DOM updates,
+  // change style, etc. to it but it won't affect rendering.
+  element.style = "...";
+  element.appendChild(...);
+
+  // Calling update() will allow rendering to go update the style & layout
```

Comment:
I think the correct thing to say here is that style and layout inside the locked element will be updated (cooperatively) but style and layout outside the locked element will not.

Reply:
cc @vmpstr to make sure

Reply:
In this case, which is lock-before-append, style, layout etc of the element and its subtree will be updated cooperatively, 
but the rest of the DOM will be updated in one go.

Reply:
Added more comments.